src/users/users.php

```diff
@@ -545,7 +557,13 @@ public function create_field_arr($fields,$post)
 		return $fld_data;
 	}
 
-	//send registration mail
+	/**
+	 * send registration mail
```

Comment:
`Send`

Reply:
The users.php file will revamp and deprecated functions will be removed from this file. I have just added a deprecated tag in it.